app/controllers/application_controller/timelines.rb

```diff
@@ -128,41 +128,7 @@ def tl_chooser
   end
 
   def getTLdata
```

Comment:
Or put another way, why don't we just call the route entrypoint miq_capacity/timeline_data to keep it consistent with the presenter naming.